lib/locales/en.ts

```diff
@@ -420,6 +429,8 @@ export const locale: Locale = {
     frances_xavier_cabrini_virgin: 'Saint Frances Xavier Cabrini, Virgin',
     francis_borgia_priest: 'Saint Francis Borgia, Priest',
     francis_de_sales_bishop: 'Saint Francis de Sales, Bishop and Doctor of the Church',
+    francis_de_sales_bishop_patron_of_lyon_clergy:
+      'Saint Francis de Sales, Bishop and Doctor of the Church, Patron of the Lyon clergy',
```

Comment:
`Lyon`: City or Diocese?

Reply:
Diocese 👍

Reply:
Then it needs to be updated. :wink:

```suggestion
      'Saint Francis de Sales, Bishop and Doctor of the Church, Patron of the Clergy of the Archdiocese of Lyon',
```

Reply:
Yes, I had only completed the key on your suggestion, but here, I simply translated literally the title from French. 

But yes, you are right, it's better this way. However, I'm tempted to leave exactly what is written in the missal in French.

Reply:
I think we should abide the following rules:

- for celebrations which have names officially translated (usually they are celebrated in that language), we should use the official celebration name;
- for celebrations which have no officially translated names:
  - we should do our best to translate the celebration name;
  - diocese/city names should be written always in full;
  - ideally, we should also add the country name after the diocese/city name (both in the key name and its value) which is not the case ATM.

Reply:
> However, I'm tempted to leave exactly what is written in the missal in French.

I absolutely agree with that! :wink: